code/modules/research/designs/autolathe_designs.dm

```diff
@@ -1221,3 +1221,10 @@
 	materials = list(/datum/material/iron = 100, /datum/material/glass = 50)
 	build_path = /obj/item/assembly/control
 	category = list("initial","Misc")
+/datum/design/razor
+	name = "Electric Razor"
+	id = "razor"
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+	materials = list(/datum/material/iron = 75)
+	build_path = /obj/item/razor
+	category = list("initial","Medical")
```

Comment:
is it really medical though

Reply:
Acording to cargo crates yes, but I guess I can move it to Misc, my problem is that misc is just really really packed and I dont wanna pack it more

Reply:
you didnt put this in techwebs its not going to be printable in medlathe, also you didnt set a departmental flag

Reply:
oh right. I just copied how other tools were put in. I intend it to be printable everywhere in medical section.

Reply:
not just in medical. Altho I can move it to autolathe only, There is only one autolathe tho so I am on the fence.

Reply:
alright. i am moving it to autolathe and service lathe. it stays in medical section tho cuz yes